src/otx/v2/adapters/torch/mmengine/mmdet/modules/models/backbones/imgclsmob.py

```diff
@@ -0,0 +1,164 @@
+
+from otx.v2.api.utils.logger import get_logger
+
+# ruff: noqa: SLF001
+
+
+def replace_activation(model: nn.Module, activation_cfg: dict) -> nn.Module:
+    """Replace activate funtion."""
+    for name, module in model._modules.items():
+        if len(list(module.children())) > 0:
+            model._modules[name] = replace_activation(module, activation_cfg)
+        if "activ" in name:
+            if activation_cfg["type"] == "torch_swish":
+                model._modules[name] = nn.SiLU()
+            else:
+                model._modules[name] = build_activation_layer(activation_cfg)
+    return model
+
+
+def replace_norm(model: nn.Module, cfg: dict) -> nn.Module:
+    """Replace norm funtion."""
+    for name, module in model._modules.items():
+        if len(list(module.children())) > 0:
+            model._modules[name] = replace_norm(module, cfg)
+        if "bn" in name:
+            model._modules[name] = build_norm_layer(cfg, num_features=module.num_features)[1]
+    return model
+
+
+def multioutput_forward(self: nn.Module, x: Tensor) -> list[Tensor]:
+    """Multioutput forward function for new model (copy from mmdet older)."""
+    outputs = []
+    y = x
+
+    last_stage = max(self.out_indices)
+    for i, stage in enumerate(self.features):
+        y = stage(y)
+        s_verbose = str(i) + " " + str(y.shape)
+        if i in self.out_indices:
+            outputs.append(y)
+            s_verbose += "*"
+        if self.verbose:
+            print(s_verbose)
+        if i == last_stage:
+            break
+
+    return outputs
+
+
+def train(self: nn.Module, mode: bool = True) -> None:
+    """Train forward function for new model (copy from mmdet older)."""
+    super(self.__class__, self).train(mode)
+
+    for i in range(self.frozen_stages + 1):
+        feature = self.features[i]
+        feature.eval()
+        for param in feature.parameters():
+            param.requires_grad = False
+
+    if mode and self.norm_eval:
+        for module in self.modules():
+            # trick: eval have effect on BatchNorm only
+            if isinstance(module, _BatchNorm):
+                module.eval()
+
+
+def init_weights(self: nn.Module, pretrained: bool = True) -> None:
+    """Init weights function for new model (copy from mmdet)."""
+    if pretrained:
+        rank, world_size = get_dist_info()
+        if rank == 0:
+            # Make sure that model is fetched to the local storage.
+            download_model(net=self, model_name=self.model_name, local_model_store_dir_path=self.models_cache_root)
+            if world_size > 1:
+                distributed.barrier()
+        else:
+            # Wait for model to be in the local storage, then load it.
+            distributed.barrier()
+            download_model(net=self, model_name=self.model_name, local_model_store_dir_path=self.models_cache_root)
+
+
+def generate_backbones() -> None:
+    """Generate backbones of pytorchcv funtion."""
+    logger = get_logger()
```

Comment:
would it be possible to create this `logger` at the top of the file so it could also be used by other functions?